packages/dapp/src/hooks/useMultisig.ts

```diff
@@ -31,14 +36,14 @@ export const useMultisig = () => {
     return !!accountInfo?.safeAddress;
   };
 
-  const isSafeSigner = !!(account?.signer?.provider as any)?.provider?.safe
-    ?.safeAddress;
+  const isSafeSigner =
+    !!(account?.signer?.provider as any)?.provider?.safe?.safeAddress || false;
```

Comment:
Will it infer the type if we make it like this? Looks cleaner this way 🙂

`Boolean((account?.signer?.provider as any)?.provider?.safe?.safeAddress)`

Reply:
Much better, thanks! Never used that approach, great you brought it up!